planning/behavior_path_planner/src/scene_module/lane_change/normal.cpp

```diff
@@ -40,13 +40,15 @@ namespace behavior_path_planner
 NormalLaneChange::NormalLaneChange(
   const std::shared_ptr<LaneChangeParameters> & parameters, LaneChangeModuleType type,
   Direction direction)
-: LaneChangeBase(parameters, type, direction)
+: LaneChangeBase(parameters, type, direction), stop_time_{0.0}
```

Comment:
if the stop_time is initialized [here](https://github.com/autowarefoundation/autoware.universe/pull/5105/files#:~:text=double%20stop_time_%3B,%7D%3B), then we don't need this line.

Reply:
Done
https://github.com/autowarefoundation/autoware.universe/pull/5105/commits/6634c6426b90c78c3f2f0471bafb42373df53a3f